projects/js-packages/publicize-components/src/components/social-image-generator/template-picker-modal/styles.module.scss

```
@@ -0,0 +1,8 @@
+.footer {
+	display: flex;
+	justify-content: flex-end;
+
+	> * + * {
```

I know this is copied over from the other CSS file, but I personally don't like magic selectors like this. Could we use
```suggestion
	> :not(:first-child) {
```

to make it a bit clearer?

---

Yeah sounds good. When we have the time we could clear up all these across social

---

Oh do we use this a lot? I think maybe this is a question of personal choice. The con to this is that it's a fair bit longer. Do you have a preference?